danger-klaxit/lib/danger_plugin.rb

```diff
@@ -119,6 +120,29 @@ def warn_for_bad_order_in_config(config_file = "config/config.yml")
     MARKDOWN
   end
 
+  def warn_for_not_updated_structure_sql
+    migration_files = (git.added_files - %w(Dangerfile)).grep(%r(db/migrate))
```

Comment:
Don't see the point of removing Dangerfile here. bad Copy/pasta 🍝  I guess?
```suggestion
    migration_files = git.added_files.grep(%r(db/migrate))
```